sagemaker_rl_agent/lib/python3.5/site-packages/markov/evaluation_worker.py

```diff
@@ -125,6 +125,10 @@ def main():
                         type=str,
                         nargs='+',
                         default=rospy.get_param("MODEL_S3_PREFIX", ["sagemaker"]))
+    parser.add_argument('--s3_endpoint_url',
+                        help='(string) S3 endpoint URL',
+                        type=str,
+                        default=rospy.get_param("S3_ENDPOINT_URL", None))                        
```

Comment:
Is `S3_ENDPOINT_URL` a default parameter? If not, could we have a safe default, for instance what value is used as an S3 endpoint when it's not provided? Will `None` evaluate to same value as before the changes?

Reply:
Yes. The Boto3 library will 'ignore' the endpoint_url parameter if it is None. Hence my attempt is to always make the parameter optional, and always with default value None to ensure that we can also use a real S3 backend if running it on EC2.